### PR TITLE
Document DecoderSample in ReadMe.md

### PR DESCRIPTION
**Pull request by TechPreacher**
Document DecoderSample in ReadMe.md in its own solution folder.

### CODE REVIEW

File: Samples/DecoderSample/ReadMe.md [hunk 1]
```diff
@@ -1,27 +1,78 @@
...
+**Make sure to choose all lowercase letters for the Module Name as the container will be unreachable otherwise!**
+
+![Decoder Sample - Edge Module](/pictures/decodersample-edgemodule.png)
+
+To activate the decoder for a LoRa device, navigate to your IoT Hub -> IoT Devices -> Device Details -> Device Twin and set the ```SensorDecoder``` value in the desired properties to: 
+
+```
+http://<decoder module name>/api/<DecoderName>
+```
+
```
Comment (ronniesa) [inline Samples/DecoderSample/ReadMe.md]: I know how it works but it is a bit confusing that you use port 8881 for testing but then the deployed version use no port (default port 80). Can you just document this part so it is more clear.

Comment (ronniesa) [reply Samples/DecoderSample/ReadMe.md]: Rest is good and clear

File: Samples/DecoderSample/ReadMe.md [hunk 2]
```diff
@@ -1,27 +1,78 @@
...
+docker run --rm -it -p 8881:80 --name decodersample <container registry>/<image>:<tag>
+````
+
+You can then use a browser to navigate to:
+
+```
+http://localhost:8881/api/DecoderValueSensor?fport=1&payload=ABCDE12345
+```
+
+If required, add credentials to access your container registry to the IoT Edge device by adding them to IoT Hub -> IoT Edge -> Your Device -> Set Modules -> Container Registry settings.
```
Comment (fbeltrao) [inline Samples/DecoderSample/ReadMe.md]: Tip: can use arrows in markdown with `&rarr;`: &rarr;

File: Samples/DecoderSample/ReadMe.md (hunk 1 above)
Comment (TechPreacher) [reply Samples/DecoderSample/ReadMe.md]: Will do! Thanks for the input!

File: Samples/DecoderSample/ReadMe.md (hunk 2 above)
Comment (TechPreacher) [reply Samples/DecoderSample/ReadMe.md]: Great!! Thanks! Will edit accordingly.

File: Samples/DecoderSample/ReadMe.md (hunk 1 above)
Comment (TechPreacher) [reply Samples/DecoderSample/ReadMe.md]: I have re-written the part on debugging on localhost as I also found out that the launchSettings.json is ignored by Git and each person will get the default config of http://localhost:5000

File: Samples/DecoderSample/ReadMe.md (hunk 2 above)
Comment (TechPreacher) [reply Samples/DecoderSample/ReadMe.md]: Done! Really looks better!

**Pull request merged**